server/user-bootstrap/index.js

```diff
@@ -36,24 +36,11 @@ const debug = debugFactory( 'calypso:bootstrap' ),
  * @returns {Promise<object>} A promise for a user object.
  */
 module.exports = function( request ) {
-	const authCookieValue = request.cookies.wordpress_logged_in;
+	const authCookieValue = request.cookies[ AUTH_COOKIE_NAME ];
 	const geoCountry = request.get( 'x-geoip-country-code' ) || '';
 	const supportSession = request.get( 'x-support-session' );
 
 	return new Promise( ( resolve, reject ) => {
-		if ( authCookieValue && supportSession ) {
-			reject(
-				new Error(
-					'Both an auth cookie and a support session were provided for bootstrap. This should not occur.'
-				)
-			);
-			return;
-		}
-		if ( ! authCookieValue && ! supportSession ) {
-			reject( new Error( 'Cannot bootstrap without an auth cookie or a support session.' ) );
-			return;
-		}
```

Comment:
This error message is still valid, isn't it? In case where both login cookie and support session headers are missing?

Reply:
Yeah, it still applies. We added the message in the initial support sessions PR when bootstrap required one or the other. Before that, we just sent the server what we had and allowed it to respond with failure.

I was on the fence about whether to keep it and thought it might be better to let the server be the judge. I can re-add it if you prefer though.

Reply:
So the idea is that when both login cookie and support session header are missing, we will send a REST request with no authentication headers at all? And let server return an error?

I'd prefer to fail early if we know for sure we don't have any auth headers to add 🙂

Reply:
No problem. I'll re-add the message. 😄